packages/@okta/vuepress-theme-default/assets/css/okta/pages/guides.scss

```diff
@@ -12,10 +12,24 @@ $big-spacing: 3rem;
 
 .guide-details { 
   display: flex;
+<<<<<<< HEAD
```

Comment:
Looks like a merge conflict - you probably aren't using the latest baseline of `guides`